tests/foreman/ui/test_rhc.py

```diff
@@ -0,0 +1,163 @@
+"""Test class for RH Cloud connector - rhc
+
+:Requirement: Remoteexecution
+
+:CaseAutomation: Automated
+
+:CaseLevel: Acceptance
+
+:CaseComponent: RHCloud-CloudConnector
+
+:Assignee: jpathan
+
+:TestType: Functional
+
+:CaseImportance: High
+
+:Upstream: No
+"""
+import pytest
+from fauxfactory import gen_string
+from wait_for import wait_for
+
+from robottelo import constants
+from robottelo.config import settings
+from robottelo.constants import DEFAULT_SUBSCRIPTION_NAME
+from robottelo.helpers import file_downloader
+from robottelo.logging import logger
+
+
+@pytest.fixture(scope='module')
+def fixture_enable_rhc_repos(module_target_sat):
+    """Enable repos required for configuring RHC."""
+    # subscribe rhc satellite to cdn.
+    module_target_sat.register_to_cdn()
+    if module_target_sat.os_version.major == 8:
+        module_target_sat.enable_repo(constants.REPOS['rhel8_bos']['id'])
+        module_target_sat.enable_repo(constants.REPOS['rhel8_aps']['id'])
+    else:
+        module_target_sat.enable_repo(constants.REPOS['rhscl7']['id'])
+        module_target_sat.enable_repo(constants.REPOS['rhel7']['id'])
+
+
+@pytest.fixture(scope='module')
+def module_rhc_org(module_target_sat):
+    """Module level fixture for creating organization"""
+    org = module_target_sat.api.Organization(
+        name=settings.rh_cloud.organization or gen_string('alpha')
+    ).create()
+    # adding remote_execution_connect_by_ip=Yes at org level
+    module_target_sat.api.Parameter(
+        name='remote_execution_connect_by_ip',
+        value='Yes',
+        organization=org.id,
+    ).create()
+    return org
+
+
+@pytest.fixture()
+def fixture_setup_rhc_satellite(request, module_target_sat, module_rhc_org):
+    """Create Organization and activation key after successful test execution"""
+    yield
+    if request.node.rep_call.passed:
+        manifests_path = file_downloader(
+            file_url=settings.fake_manifest.url['default'], hostname=module_target_sat.hostname
+        )[0]
+        module_target_sat.cli.Subscription.upload(
```

Comment:
@jameerpathan111 Can you use cloned manifest instead of the original one?
@synkd I look forward to having manifester in robottelo! This is exactly one of the tests where it would fit, right?

Reply:
Well, I can't and that's why this workaround. And it doesn't affect any other tests.

Reply:
I do not want to block your PR on this, but this test is, as you say, gives the Satellite with the original manifest set up to iqe tests.
Who will control what the tests do with the Sat instance? How do we ensure that no one runs manifest refresh there?
I want to avoid manifest issues like #9546 tried to solve.
@JacobCallahan What is your opinion?

Reply:
@ogajduse Yes, absolutely. Test modules such as this one will be able to import Manifester, instantiate a Manifester object, then use the `get_manifest` method to generate a manifest. The contents of the manifest can be controlled via Manifester's settings file.

Reply:
I have replied for this in the GChat, just extending my thoughts here:

@jameerpathan111 For now we should be able to continue with the original manifest as the RHC test case in robottelo is not refreshing the manifest and there is no impact on other tests!  But we need to check with @Ichimonji10 if any of the iqe-remediations test cases doing a manifest refresh when we go for stage 3 in RHC E2E pipeline!

If not refreshing from iqe-rem automation test cases, then we are good for using the original manifest until we have manifester integrated with robottelo from @synkd but if iqe runs this pipeline then they should pass their own manifest in the job so that they can play with it! @jameerpathan111 We need to do such provisioning in state1/2 of E2E pipeline before we have stage 3 in place!

So for now ACK from me !!

Reply:
I checked with Jeremy and he said `The Satellite is assumed to be totally pre-configured. The automated tests do not mutate the satellite configuration in any way.`.

Reply:
Not a concern from my side anymore. Feel free to resolve this thread.